lib/internal/errors.js

```diff
@@ -111,7 +111,23 @@ module.exports = exports = {
 //
 // Note: Please try to keep these in alphabetical order
 E('ERR_ARG_NOT_ITERABLE', '%s must be iterable');
+E('ERR_ARG_OUT_OF_BOUNDS',
+  (name) => `The "${name}" argument is out of bounds`);
 E('ERR_ASSERTION', (msg) => msg);
+E('ERR_BUFFER_ENCODING_PROVIDED',
+  'If encoding is specified then the first argument must be a string');
+E('ERR_BUFFER_INVALID_CONCAT_LIST',
+  'The "list" argument must be an Array of Buffer or Uint8Array instances');
```

Comment:
I'm not sure if these types would only fit for buffers and therefore I recommend to remove the `BUFFER` part from the name.